### PR TITLE
updated documentation for 3D Tiles styling

### PR DESCRIPTION
**Pull request by hanbollar**
Fixes [#6358](https://github.com/AnalyticalGraphicsInc/cesium/issues/6358)

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Signed CLA is on file.

@hanbollar, thanks for the pull request! Maintainers, we have a signed CLA from @hanbollar, so you can review this at any time.

:warning: I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

**Comment by hanbollar** [comment]
@lilleyse Let me know if this is what you were looking for!

File: Source/Scene/Cesium3DTileStyle.js [hunk 1]
```diff
@@ -237,6 +237,8 @@ define([
          * The expression must return or convert to a <code>Boolean</code>.
          * </p>
          *
+         * Applicable to all tile formats.
```
Comment (lilleyse) [inline Source/Scene/Cesium3DTileStyle.js]: To be consistent with the other descriptions, adjust to: `This expression is applicable to all tile formats`
Wrap with `<p> </p>` like the others.
Add this note to `color` as well.

File: Source/Scene/Cesium3DTileStyle.js [hunk 2]
```diff
@@ -425,6 +427,9 @@ define([
          * <p>
```
Comment (lilleyse) [inline Source/Scene/Cesium3DTileStyle.js]: Above - `pointSize` should mention that is only applicable to point features in a Vector tile or Point Cloud tile.

Comment (lilleyse) [reply Source/Scene/Cesium3DTileStyle.js]: > as a note - in the issue you mentioned pointSize is used by both but in the linked docs pointSize is only used for point features - for now i dont have the "This expression is only applicable to point features in a Vector tile" line for it - lmk if you want me to add it into the docs there as well

I didn't link to it, but `pointSize` is also used for Point Cloud tiles [here](https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/Styling#point-cloud)

**Comment by hanbollar** [comment]
@lilleyse updated

File: Source/Scene/Cesium3DTileStyle.js [hunk 3]
```diff
@@ -236,6 +236,11 @@ define([
...
          * The expression must return or convert to a <code>Boolean</code>.
          * </p>
+         * <p>
+         * This expression is applicable to all tile formats.
+         * </p>
+         *
+         * Applicable to all tile formats.
```
Comment (lilleyse) [inline Source/Scene/Cesium3DTileStyle.js]: Remove this line, now a duplicate.

Comment (hanbollar) [reply Source/Scene/Cesium3DTileStyle.js]: woops - i must have missed that from the first time around. fixed now.

**Comment by lilleyse** [comment]
Thanks @hanbollar!

**Pull request merged**